core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/models.scala

```diff
@@ -32,3 +32,7 @@ object AppContext {
     )
 
 }
+
+final case class WorkspaceId(value: String) extends AnyVal
+
+final case class CloudContext(value: String) extends AnyVal
```

Comment:
Almost feels like this should be slightly more intelligent than a simple value class. It seems a little pointless, if the intent is to wrap a GoogleProject AND an Azure string identifier, to represent them with more-or-less a string wrapper